src/sass/disability-benefits.scss

```diff
@@ -9,3 +9,12 @@
 // @import "modules/m-omb-info";
 @import "modules/m-form-confirmation";
 
+.dashed-underline {
+  text-decoration: none;
+  border-bottom: 1px dashed;
+
+  &:hover {
```

Comment:
Because `.va-button-link` also has `&:hover`, this is needed.

Reply:
Is this something we should do everywhere?

Reply:
:man_shrugging: 
I can put it in `base/_m-utils.scss`.